packages/backend/drizzle.config.ts

```diff
@@ -7,7 +7,7 @@ export default defineConfig({
   out: './src/db/migrations',
   dialect: 'postgresql',
   dbCredentials: {
-    url: 'postgresql://postgres:letmein@localhost:5432/Develeb-platform',
+    url: process.env.DATABASE_URL!,
```

Comment:
why the !

Reply:
The type of the process.env.DATABASE_URL is string|undefined which is generating the error saying url can only be a string and not string|undefined. The ! serves to tell the compiler that this value cannot be undefined.